topics/ecology/tutorials/phylogeny-data-prep/tutorial.md

```diff
@@ -59,12 +59,12 @@ The resulting dataset is ready to be used for phylogenetic reconstruction.
 # Get data
 For this training we will use a subset of the genome (chromosome 5) from four strains of *S. cerevisiae*. The GenBank annotated sequenced were produced using 'funannotate predict annotation' (Galaxy Version 1.8.9+galaxy2) on the nucleotide sequences sequences.
 
-> ### {% icon hands_on %} Hands-on: Data upload
+> <hands-on-title>Data upload</hands-on-title>
 >
 > 1. Create a new history for this tutorial
-> 2. Import the files from [Zenodo]({{ https://zenodo.org/record/6610704#.Ypn3FzlBw5k }}) or from
->    the shared data library (`GTN - Material` -> `{{ ecology }}`
->     -> `{{ phylogenetic-data-prep }}`):
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}) or from
+>    the shared data library (`GTN - Material` -> `ecology`
+>     -> `phylogenetic-data-prep`):
```

Comment:
```suggestion
>     -> Preparing genomic data for phylogeny reconstruction ):
```
Since users will see the tutorial title listed

(or we can use `{{page.topic}}` and `{{page.title}}` here to make it rename/move-proof)